### ISSUE TITLE
Complete failure.

### ISSUE THREAD

**Issue by PolarDog23** [NightscoutFoundation/xDrip#99]
For some unknown reason, my Moto Z Force no longer recognizes the bluetooth transmitter.  It has been two days now. Transmitter just stopped working.  I've tried two different (new) transmitters thinking that one was bad. No luck. 
Transmitter appears to be connecting to the Dexcom receiver, so I may be forced to suffer with another device again, but I would rather not.   
I see now that that the Beta is over and I have since downloaded the latest Nightscout apk. Same result. What can I do?

**Comment by jamorham** [comment]
Is this with a G5? Which xDrip+ version are you running? Use System Status menu item to find out

**Comment by PolarDog23** [comment]
I finally got it working late last night. I'm on version 2017.03.27.

For whatever reason, the phone wouldn't recognize the transmitter via bt.

It finally did (guess the phone got tired of my persistence), but then when
I selected the transmitter from the list, a screen would display asking for
the sn of the Dexcom receiver I was using (?)  After clicking out of that
screen, I saw that the collection method had automatically switched to G4.
After several tries more, it finally stayed on G5.

I think I'm good for now. I guess a lot has happened since I first started
using xDrip that I missed. I wasn't ready for the steep curve, and the
changes to the transmitters that Dexcom made certainly didn't help.

There was a very nice and patient person that helped me last night and for
her I am thankful.

And thank you!

Eric




On Mar 29, 2017 7:31 AM, "Jamorham" <notifications@github.com> wrote:

Is this with a G5? Which xDrip+ version are you running? Use System Status
menu item to find out

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/NightscoutFoundation/xDrip/issues/99#issuecomment-290075087>,
or mute the thread
